scipy/stats/_continued_fraction.py

```diff
@@ -300,7 +300,9 @@ def func(n, *args):
 
     xp = array_namespace(fs_a[0], fs_b[0], *args)
 
-    shape = xp.broadcast_shapes(shape_a, shape_b)
+    # TODO: broadcast_shapes is not part of the Array API (but maybe it should be ...)
+    # temporarily use numpy
+    shape = np.broadcast_shapes(shape_a, shape_b)
```

Comment:
I don't think dask has ``broadcast_shapes`` and it is not something that seems to be part of the 2023.12 array API spec.

It does seem useful, though.
Should we make an issue to add it to array API or turn it into a helper somewhere in scipy?